### DISCUSSION TITLE
ORC error

### DISCUSSION THREAD

**Discussion by SiNaPsEr0x** [docker-library/postgres#1087] [General]
on armhf with repository arm32v7 its all ok.. but..

```
ORC error: No callback manager available for armv7-alpine-linux-musleabihf
database_1   | ORC error: No callback manager available for armv7-alpine-linux-musleabihf
database_1   | 2023-05-17 11:04:25.206 UTC [110] LOG:  invalid length of startup packet
```

why???

**Comment by tianon** [comment]
Unfortunately, we do not have the bandwidth to provide in-depth integration/deployment/environment debugging or support here; these sorts of questions/requests would be more appropriately posted to a dedicated support forum, such as [the Docker Community Slack](https://dockr.ly/comm-slack), [Server Fault](https://serverfault.com/help/on-topic), [Unix & Linux](https://unix.stackexchange.com/help/on-topic), or [Stack Overflow](https://stackoverflow.com/help/on-topic).

**Comment by SiNaPsEr0x** [comment]
Resolve in QNAP 

jit = off
at file -> /share/Container/container-station-data/lib/docker/volumes/teslamate_teslamate-db/postgresql.conf
